modules/tinymce/src/core/test/ts/browser/content/InsertListTest.ts

```diff
@@ -1,48 +0,0 @@
-import { describe, it } from '@ephox/bedrock-client';
-import { assert } from 'chai';
-
-import DOMUtils from 'tinymce/core/api/dom/DOMUtils';
-import DomParser from 'tinymce/core/api/html/DomParser';
-import AstNode from 'tinymce/core/api/html/Node';
-import Schema from 'tinymce/core/api/html/Schema';
-import * as InsertList from 'tinymce/core/content/InsertList';
-
-describe('browser.tinymce.core.content.InsertListTest', () => {
```

Comment:
Why was this deleted? Based on what it's doing it shouldn't have been removed.

- [ ] Revert or provide a good reason as to why this was deleted

Reply:
It's back (just skipped with some more `any` types and TODOs)